src/chown/chown.rs

```diff
@@ -169,8 +169,11 @@ pub fn uumain(args: Vec<String>) -> i32 {
             }
         }
     }
+    let using_ref = matches.opt_str("reference").is_some();
     let mut files = matches.free;
-    files.remove(0);
+    if !using_ref {
```

Comment:
It would probably make more sense to move this into the `if` body above.

Reply:
I've done this, although it looks a bit messy as I had to work around mutably borrowing `matches.free` too early.